generated/nidaqmx/_task_modules/channels/channel.py

```diff
@@ -19,16 +19,18 @@ class Channel:
     """
     __slots__ = ['_handle', '_name', '__weakref__']
 
-    def __init__(self, task_handle, virtual_or_physical_name):
+    def __init__(self, task_handle, virtual_or_physical_name, interpreter):
         """
         Args:
             task_handle (TaskHandle): Specifies the handle of the task that
                 this channel is associated with.
             virtual_or_physical_name (str): Specifies the flattened virtual or
                 physical name of a channel.
+            interpreter: Specifies the interpreter instance assigned.
```

Comment:
In all of these docstrings, when referring to the interpreter:
- Specify the type: `interpreter (BaseInterpreter): ...`
- Remove "assigned". "Specifies the interpreter instance." is perfectly clear without it.